modules/apps/object/object-api/src/main/java/com/liferay/object/scope/ObjectScopeProvider.java

```diff
@@ -25,6 +25,8 @@ public long getGroupId(HttpServletRequest httpServletRequest)
 
 	public String[] getRootPanelCategoryKeys();
 
+	public Long getValidGroupId(long companyId, String scopeKey);
```

Comment:
@devpedrotavares why did you decide to use `Long` instead of the primitive type `long`?

Reply:
Because it can return `null`